applovin_max/lib/src/ad_classes.dart

```diff
@@ -65,13 +74,16 @@ class MaxAd {
 
     return MaxAd(
       json['adUnitId'] as String,
+      json['adFormat'] as String,
       json['adViewId'] as AdViewId?,
       json['networkName'] as String,
+      json['networkPlacement'] as String,
       double.tryParse(json['revenue']?.toString() ?? '0.0') ?? 0.0,
       json['revenuePrecision'] as String,
       json['creativeId'] as String,
       json['dspName'] as String,
       json['placement'] as String,
+      double.tryParse(json['latencyMillis']?.toString() ?? '0.0') ?? 0.0,
```

Comment:
Hmm I think if we keep millis we will need to use int instead of double

Reply:
Updated to use integer.   Please review again.